### PR TITLE
FIX #5324

### LINKED ISSUE #5324: vars/args display 0 in afi and afll and add in aflj
vars/args display 0 in `afi` and `afll` and `aflj`

### PR DESCRIPTION
**Pull request by oddcoder**

### CODE REVIEW

**Comment by Maijin** [comment]
Missing aflj fix 😃

File: libr/core/anal.c [hunk 1]
```diff
@@ -1697,13 +1697,15 @@ static int fcn_print_legacy(RCore *core, RAnalFunction *fcn) {
...
 
 	if (fcn->type == R_ANAL_FCN_TYPE_FCN || fcn->type == R_ANAL_FCN_TYPE_SYM) {
-		r_cons_printf ("\n vars: %d", r_list_length (fcn->vars));
-		r_list_foreach (fcn->vars, iter, vari) {
-			char *s = r_anal_type_to_str (core->anal, vari->type);
-			r_cons_printf ("\n  %s %s @ 0x%02x", s, vari->name, vari->delta);
-			free (s);
-		}
-		r_cons_printf ("\n diff: type: %s",
+		int args_vars_count = r_anal_var_count (core->anal, fcn, 'b') +
```
Comment (alvarofe) [inline libr/core/anal.c]: I prefer this that that long line
```c
int args_vars_count = r_anal_var_count()
args_var_count += r_anval_var_count()
args_var_count += r_anval_var_count()
```

**Comment by Maijin** [comment]
Also, why removing local and argument?

**Comment by Maijin** [comment]
`var, arg, nargs, locals, args` :/
What about `nvars, nargs, var, arg` ? or `nlocal, nargs, local, arg` ?

**Comment by oddcoder** [comment]
um I have no idea what are these but imho vars / locals / nvars / nlocals are the same thing ?
perhaps we should ask the guy who created them first I am not sure what does the n means ? I just kept it

**Comment by alvarofe** [comment]
I understand that `nargs` means `number of args`

**Comment by Maijin** [comment]
Yep but the questionn is more about `local` vs `var` terminology

**Comment by alvarofe** [comment]
I prefer local than var

**Comment by Maijin** [comment]
Same here.

**Comment by Maijin** [comment]
@oddcoder `nlocal, nargs, local, arg` should be used then

**Comment by oddcoder** [comment]
what is locals and args ?

**Comment by XVilka** [comment]
*"Locals - are people living in the some area, you're talking about."*
Err, wrong definition, sorry:
*"Locals - local variables"*
*"Args - function arguments, in some countries also known as 'parameters'".*

**Comment by radare** [comment]
> On 17 Jul 2016, at 22:57, Anton Kochkov <notifications@github.com> wrote:
> 
> "Locals - are people living in the some area, you're talking about."

Lol

**Comment by radare** [comment]
Yeah, var doesnt species any scope. Only that it is not const. Local means relative to stack frame and global is absolute. Const and var is related to mutability

**Comment by Maijin** [comment]
I still see `var` used for variable name and comments like:

`int var_count ` or `//var: type = 0` etc...

**Comment by oddcoder** [comment]
would that be a problem ?

**Comment by Maijin** [comment]
Well, it's inconsistent and in case of someone wants to contribute that ain't nice 👎

**Comment by oddcoder** [comment]
well currently... It just happen that local => local variables , args => regular args, vars => means them both

**Pull request merged**